sdk/src/wallet/update.rs

```diff
@@ -69,7 +69,8 @@ where
                     wallet_data.unspent_outputs.remove(&output_id);
                     // Update spent data fields
                     if let Some(output_data) = wallet_data.outputs.get_mut(&output_id) {
-                        output_data.metadata.set_spent(true);
+                        // TODO wat do?
```

Comment:
Guess we just don't update it here

Reply:
Is that not going to cause issues somewhere else? Do we rely on this or it's purely informative ?

Reply:
Let me create an issue and update the TODO then